libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -46,6 +46,17 @@ ion-modal.kirby-overlay {
     }
   }
 
+  &.interact-with-background {
+    display: table;
```

Comment:
This was the best way I found to make the parent fit child content

Reply:
Let's talk about this ;)

Reply:
Now the ion-modal is positioned to fit content

Reply:
Using js